### PR TITLE
CHT: Translated some more strings, fixes #160

### LINKED ISSUE #160: Issue with Traditional Chinese
STR_3172 should be:
```
STR_3172    :The following object must be selected first: {STRING}
```
Please pay attention, @daihakken . You shouldn't remove parameters. Also, you seem to have based your last version on US English. This is wrong, you should base it on UK English, as is clearly stated in the readme. Here is a link that is always up to date: https://github.com/OpenRCT2/OpenRCT2/blob/develop/data/language/english_uk.txt

### PR DESCRIPTION
**Pull request by daihakken**
@Gymnasiast I think the issue should be fixed now :-P

### CODE REVIEW

File: data/language/chinese_traditional.txt [hunk 1]
```diff
@@ -3777,16 +3772,33 @@ STR_5436    :Edit Title Sequences...
...
+STR_5449    :減慢遊戲速度
+STR_5450    :增快遊戲速度
+STR_5451    :開啟密技視窗
 STR_5452    :Toggle visibility of toolbars
+STR_5453    :Select another ride
+STR_5454    :去除FPS上限
+STR_5455    :啟用沙盒模式
+STR_5456    :去除阻礙檢查
+STR_5457    :去除支撐限制
+<<<<<<< HEAD
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: You should remove this line. (It was an error in English_uk.txt.)

File: data/language/chinese_traditional.txt [hunk 2]
```diff
@@ -3777,16 +3772,33 @@ STR_5436    :Edit Title Sequences...
...
+STR_5458    :順時計旋轉
+STR_5459    :逆時計旋轉
+STR_5460    :Rotate view anti-clockwise
+STR_5461    :Set guests' parameters
+STR_5462    :{CURRENCY}
+STR_5463    :目標: 盡情玩吧!
+STR_5464    :一般
+STR_5465    :氣候
+STR_5466    :員工
+=======
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: You should remove this line and the one that follows it. (It was an error in English_uk.txt.)

File: data/language/chinese_traditional.txt [hunk 3]
```diff
@@ -1709,7 +1709,7 @@ STR_1705    :{SMALLFONT}{BLACK}Sack this staff member
 STR_1706    :{SMALLFONT}{BLACK}Move this person to a new location
 STR_1707    :Too many staff in game
 STR_1708    :{SMALLFONT}{BLACK}Set patrol area for this staff member
-STR_1709    :解僱員工
+STR_1709    :Sack staff
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: Did you intend to remove this translation?

File: data/language/chinese_traditional.txt [hunk 4]
```diff
@@ -1463,12 +1463,12 @@ STR_1459    :Track style
...
 STR_1461    :{SMALLFONT}{BLACK}'O' shaped enclosed track
 STR_1462    :Too steep for lift hill
-STR_1463    :遊客
+STR_1463    :Guests
 STR_1464    :Helix up (small)
 STR_1465    :Helix up (large)
 STR_1466    :Helix down (small)
 STR_1467    :Helix down (large)
-STR_1468    :員工
+STR_1468    :Staff
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: Did you intend to remove this translation?

File: data/language/chinese_traditional.txt [hunk 5]
```diff
@@ -1463,12 +1463,12 @@ STR_1459    :Track style
 STR_1460    :{SMALLFONT}{BLACK}'U' shaped open track
 STR_1461    :{SMALLFONT}{BLACK}'O' shaped enclosed track
 STR_1462    :Too steep for lift hill
-STR_1463    :遊客
+STR_1463    :Guests
```
Comment (Gymnasiast) [inline data/language/chinese_traditional.txt]: Did you intend to remove this translation?

File: data/language/chinese_traditional.txt (hunk 2 above)
Comment (daihakken) [inline data/language/chinese_traditional.txt]: I think str_5464 is not an error, and I deleted the last 2 line from the text document.

File: data/language/chinese_traditional.txt (hunk 5 above)
Comment (daihakken) [inline data/language/chinese_traditional.txt]: thanks, didn't notice about it!

**Pull request merged**